libs/ledger-live-common/src/wallet-api/converters.ts

```diff
@@ -71,6 +76,12 @@ export function currencyToWalletAPICurrency(
   currency: WalletAPISupportedCurrency,
 ): WalletAPICurrency {
   if (currency.type === "TokenCurrency") {
+    // Log currency if delisted
+    if (currency.delisted) {
+      console.warn(
+        `Warning: converting a delisted token currency ${currency.id} (${currency.name}) to WalletAPI format.`,
+      );
+    }
```

Comment:
I need to remove this by handling them in currency.list and other fetching we do, probably something we just want to filter at the request level